src/Aspire.Hosting.Azure/Bicep/AzureBicepResource.cs

```diff
@@ -77,15 +77,12 @@ public BicepTemplateFile GetBicepTemplateFile(string? directory = null, bool del
                     ? path
                     : Path.Combine(directory, $"{TemplateResourceName.ToLowerInvariant()}");
 
-                if (!File.Exists(path))
-                {
-                    // REVIEW: We should allow the user to specify the assembly where the resources reside.
-                    using var resourceStream = GetType().Assembly.GetManifestResourceStream(TemplateResourceName)
-                        ?? throw new InvalidOperationException($"Could not find resource {TemplateResourceName} in assembly {GetType().Assembly}");
+                // REVIEW: We should allow the user to specify the assembly where the resources reside.
```

Comment:
For future consideration. I'm not sure I like the embedded resource approach. I think it makes things less discoverable. I wonder what it might be like if the Nuget package dropped a bunch of loose Bicep files into the referencing project? That way people could easily hack on them, but also potentially make changes to the Bicep templates more discoverable/reviewable?